site/docs/v1/tech/identity-providers/samlv2/okta-saml.adoc

```diff
@@ -0,0 +1,107 @@
+
+image::identity-providers/okta-saml/okta-saml-new-idp.png[new identity provider,width=1200,role=bottom-cropped]
+
+Provide a [field]#Name#, for example, `OKTA SAML`. For the [field]#IdP endpoint#, paste the [field]#Identity Provider Single Sign-On URL# from the Okta [breadcrumb]#SAML setup instructions# page. For the verification key, select from the dropdown the name of the key you just created in the Key Master. Optionally, change the [field]#Button text# to `Login with Okta`.
+
+image::identity-providers/okta-saml/okta-saml-new-idp-settings.png[new idp settings,width=1200]
+
+Make sure your application has the [uielement]#Enabled# switch toggled. Then, navigate to the [breadcrumb]#Options# tab and supply the names that you chose for the [field]#user.Id# and [field]#user.email# attribute statements in Okta.
+
+image::identity-providers/okta-saml/okta-saml-claim-options.png[claim options,width=1200]
+
+Hit [uielement]#Save# and [uielement]#View# the identity provider you just created. Scroll down to the [breadcrumb]#SAML v2 Integration details# section and record these values so you can supply them to Okta.
+
+image::identity-providers/okta-saml/okta-saml-integration-details.png[integration details,width=1200]
+
+If you receive a warning after saving the identity provider, navigate to [breadcrumb]#Settings -> System# and add your Okta URL to the [field]#Allowed origins# list.
+
+image::identity-providers/okta-saml/okta-saml-cors-warning.png[cors warning,width=1200,role=bottom-cropped]
+
+image::identity-providers/okta-saml/okta-saml-cors-settings.png[cors settings,width=1200]
+
+Return to your Okta dashboard. Navigate to [breadcrumb]#Applications -> Your Application# and edit the [breadcrumb]#SAML Settings#.
+
+image::identity-providers/okta-saml/okta-saml-edit-saml-settings.png[edit SAML settings,width=1200,role=bottom-cropped]
+
+Click [uielement]#Next# to get to the [breadcrumb]#Configure SAML# tab. Replace the [field]#Single sign-on URL# with the [field]#Callback URL (ACS)# integration detail from FusionAuth. Replace the [field]#Audience URI (SP Entity ID)# with the [field]#Issuer# integration detail from FusionAuth. Click [uielement]#Next# and [uielement]#Finish# to save these settings.
+
+image::identity-providers/okta-saml/okta-saml-settings-from-fusionauth.png[settings from fusionauth,width=1200]
+
+== Testing the Login
+
+To test, navigate to the applications page in FusionAuth. Click on the [uielement]#View# icon (magnifying glass) next to the application you enabled Okta login on and copy the [field]#OAuth IdP login URL# address. Navigate to this address. You should see a [uielement]#Login with Okta# option on your app's sign-in page:
+
+image::identity-providers/okta-saml/okta-saml-result.png[result,width=1200,role=bottom-cropped]
+
+Click the [uielement]#Login with Okta# button. Test logging in with the username and password, as well as the 2FA method you selected when you configured your account on Okta.
```

Comment:
Not sure why 2fa is set up for okta? How does that help the tutorial?

Reply:
Okta enforces using a form of 2FA when creating a new account, so we thought it best to mention it in the guide. 

<img width="428" alt="Screenshot 2023-03-09 at 16 22 51" src="https://user-images.githubusercontent.com/4689280/224054251-21b443e3-b201-4a0d-b474-dba311dc3569.png">